### PR TITLE
Update make-a-plant-watering-robot.md

### PR DESCRIPTION
**Pull request by sguequierre**
* Add more detailed adding module instructions

### CODE REVIEW

File: docs/tutorials/projects/make-a-plant-watering-robot.md [hunk 1]
```diff
@@ -357,9 +357,10 @@ You can add a module from the Viam registry directly from your robot’s Configu
 
 Add the mcp300x-adc-sensor module to your robot in 3 steps:
 
-1. Go to your robot and from the create component section, search mcp300x.
-2. Add this module to your robot.
-3. Add the necessary attributes.
+1. Go to your robot's **Config** tab. Select **create component**.
```
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:360]:
```suggestion
1. Go to your robot's **Config** tab. Select **Create component**.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md [hunk 2]
```diff
@@ -371,9 +366,20 @@ Add the mcp300x-adc-sensor module to your robot in 3 steps:
...
 If you wire and configure another sensor, such as a temperature sensor on channel 1, you can add the sensor to the `"channel_map"` and get a reading from it.
-{{< alert title="Info" color="info" >}} If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor). {{< /alert >}}
+
+{{< alert title="Info" color="info" >}}
+If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor).
+{{< /alert >}}
+
+{{% alert title="Tip" color="tip" %}}
+
+Now that you have set up your robot and are able to control your motor, you can put the suction tube of your pump into the water cup, and the output tube into the plant!
```
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:380]: This needed to be moved as it was in a weird place, before finishing setting up the robot w/ the module

File: docs/tutorials/projects/make-a-plant-watering-robot.md [hunk 3]
```diff
@@ -335,13 +335,7 @@ Now, if you navigate to your robot's **Control** tab, you should be able to cont
...
 ![Creation of a pump motor in the Viam app config builder.](/tutorials/plant-watering-pi/pump-motor-control.png)
 
-{{% alert title="Tip" color="tip" %}}
-
-Now that you have set up your robot and are able to control your motor, you can put the suction tube of your pump into the water cup, and the output tube into the plant!
-
-{{% /alert %}}
-
-### Configure the ADC as a module from the registry
+#### Configure the ADC as a module from the registry
```
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:338]: Moved down one header block to make it part of "setting up your robot in the Viam app", was not clear before

File: docs/tutorials/projects/make-a-plant-watering-robot.md (hunk 2 above)
Comment (npentrel) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:380]: I think that's a good call but I think we can put this without an alert (it's visually not nice to have multiple alerts next to each other)

**Review by npentrel — APPROVED** [review]

File: docs/tutorials/projects/make-a-plant-watering-robot.md (hunk 2 above)
Comment (sguequierre) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:380]: Ok, sounds good removing

**Review by mestcihazal — APPROVED** [review]
thanks for the changes! i am also working on adding a full code to this tutorial so hopefully within this week it will be done :)

**Pull request merged**